website/content/docs/agent-and-proxy/proxy/caching/static-secret-caching.mdx

```diff
@@ -0,0 +1,140 @@
+with updated secrets.
+
+An example policy snippet that would grant access static secret (KVv1 and KVv2) events is given below:
+
+   ```hcl
+   path "sys/events/subscribe/kv*" {
+       capabilities = ["read"]
+   }
+   ```
+
+This functionality enables Proxy to receive updates as soon as a secret gets changed, reducing staleness in
+the cache, meaning Proxy only checks for a secret update upon receiving notification that the secret was updated.
+
+Additionally, tokens accessing static secrets in the cache will need the ability to access [`sys/capabilies-self`](/vault/api-docs/system/capabilities-self)
+with the `update` capability. This capability allows a client to know what capabilities have on a current path, and allows
+a token to find out capabilities on multiple paths with a single request, instead of requiring the token to reach out to
+each explicitly and see if it gets a `403` response. This is used to efficiently maintain up-to-date access management
+on the cache. If a token has its access to a KV secret revoked, this will allow Proxy to know, so that it can prevent
+that token from accessing the cached secret.
+
+This permission is provided in the [`default` policy attached by default to all Vault tokens](/vault/docs/concepts/policies#default-policy),
+and, unless this has been modified, will be attached by default to all tokens. For completeness, or in case you wish to
+apply this permission in a way other than via the `default` policy, the policy snippet required for this functionality
+is as follows:
+
+   ```hcl
+    path "sys/capabilities-self" {
+        capabilities = ["update"]
+    }
+   ```
+
+The period at which capabilities for each token using Proxy will be checked is configured with the `static_secret_token_capability_refresh_interval`
+configuration, described below. The default is to check per token every five minutes. For a setup where the `api_proxy`
+has been configured with `use_auto_auth_token` as `true` or `force`, and all requests going through Proxy will be
+using this sole token, this will be a single request to Vault per this interval, no matter how many secrets are
+being cached.
+
+## Functionality
+
+When enabled, Vault Proxy will cache `GET`s to KVv1 and KVv2 endpoints. When a `GET` is made to a KV secret,
+we will cache the response in Proxy's cache. Once the response is cached, subsequent `GET`s to this secret will
+be served from the cache instead of being forwarded to Vault, as long as this client has access to this secret
+in Vault.
+
+If a token requests a KV secret, we store in the cache that this token has the capability to access a secret.
+Therefore, subsequent requests by the same token will be able to access this secret from the cache. For a second
+token to access the same secret, it must first complete a successful `GET` to the secret before its permission
+to access the secret is cached, and its subsequent requests can be served from the cache instead of Vault itself.
```

Comment:
This reads a little confusingly to me, since the token is cached as part of the client making the request, i.e. two requests are not required and it might sound like tokens are not part of client requests somehow? I think your version is still better, and I've updated it, but I wonder if we can still make it clearer